libvisual/configure.ac

```diff
@@ -396,15 +396,12 @@ fi
 
 if test  x$lv_tool_enabled = xyes -o x$examples = xyes ; then
   # Find SDL 1.x.x
-  AC_PATH_TOOL(SDL_CONFIG, [sdl-config])
-  if test x$SDL_CONFIG = x ; then
+  PKG_CHECK_MODULES([SDL], [sdl >= 1.2.0], [], [
        AC_MSG_ERROR([*** sdl-config not found.
                        If you have installed from binaries, probably you have not
```

Comment:
The error message should be updated as well.

Reply:
@kaixiong good catch, done